explorer/interpreter/type_checker.cpp

```diff
@@ -987,8 +1003,13 @@ auto TypeChecker::ArgumentDeduction::Deduce(Nonnull<const Value*> param,
       if (arg->kind() != Value::Kind::PointerType) {
         return handle_non_deduced_type();
       }
-      return Deduce(&cast<PointerType>(*param).type(),
-                    &cast<PointerType>(*arg).type(),
+      const auto& param_pointed = cast<PointerType>(param)->type();
```

Comment:
We should probably be consistent about whether we say "pointed" or "pointee". I prefer "pointee" because it's clearly a noun, whereas "pointed" could be read as a verb (e.g. maybe `param_pointed` is a `bool` that indicates whether a parameter pointed to something).

Reply:
Makes sense 👍 , the rest of the code already uses pointee